R/arc-select.R

```diff
@@ -170,14 +211,17 @@ collect_layer <- function(x, n_max = Inf, token = Sys.getenv("ARCGIS_TOKEN"), ..
   suppressMessages(
     n_feats <- httr2::resp_body_json(
       httr2::req_perform(
-        httr2::req_url_query(n_req, f = "pjson")
+        httr2::req_url_query(n_req, f = "pjson"),
+        error_call = error_call
       ), check_type = FALSE
     )[["count"]]
   )
 
   if (is.null(n_feats)) {
-    stop(
-      "Could not determine the number of features.\n  Is your `where` statement invalid?"
+    cli::cli_abort(
+      c("Can't determine the number of features for {.arg x}.",
+      "*" = "Check if your {.arg where} statement is invalid."),
```

Comment:
If we're adjusting to use positive langauge we need to change `invalid` to `valid`